src/pymedphys/dcm/_level2/header_tweaks.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2019 Cancer Care Associates
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+from copy import deepcopy
+
+import pydicom
+
+from ...libutils import get_imports
+
+from .._level1.dcmcreate import dcm_from_dict
+
+IMPORTS = get_imports(globals())
+
+
+def adjust_machine_name(dcm, new_machine_name):
+    """Change the machine name within the DICOM header
+    """
+
+    new_dcm = deepcopy(dcm)
+
+    for beam in new_dcm.BeamSequence:
+        beam.TreatmentMachineName = new_machine_name
+
+    return new_dcm
+
+
+def adjust_machine_name_cli(args):
+    dcm = pydicom.read_file(args.input_file, force=True)
+    new_dcm = adjust_machine_name(dcm, args.new_machine_name)
+
+    pydicom.write_file(args.output_file, new_dcm)
+
+
+def delete_sequence_item_with_matching_key(sequence, key, value):
+    new_sequence = deepcopy(sequence)
+
+    for i, item in reversed(list(enumerate(sequence))):
+        try:
+            if value == getattr(item, key):
+                new_sequence.pop(i)
+        except AttributeError:
+            pass
+
+    return new_sequence
+
+
+def adjust_rel_elec_density(dcm, adjustment_map):
```

Comment:
Monaco doesn't export the electron densities of a couch. I needed to create a DICOM server to stand between Monaco and DoseCHECK and append the electron densities of the couch to the structure set.